docs/DeveloperGuide.md

```diff
@@ -182,12 +183,49 @@ This section describes some noteworthy details on how certain features are imple
 
 ##### Implementation
 
+#### 2.3 Show event
+
+#### Implementation
+
+The following is a detailed explanation of the operations which take place when the `showevents` command is called
+
+1. After successfully parsing of user input the `ShowEventCommand#exeucte(Model model)` method is called.
+2. Since all events are stored in a `FilterList`, a `PREDICATE_SHOW_ALL_EVENTS` is passed to the list so that it now contains all the events in the addressbook.
```

Comment:
1. Typo. Should be `FiteredList`.
2. Explanation of `PREDICATE_SHOW_ALL_EVENTS` could also be helpful.
3. Explanation of how the GUI switches the event would also be useful. 

Also, note that for this week, only 1 feature is needed to be documented, and needs to be 1+ page. Might want to write more, and in more detail for this one.

Reply:
Actually, it was written at least 1 feature needs to be documented, and each person needs to document at least 1 page. Nevertheless, I will be adding more details as suggested.